### DISCUSSION TITLE
How to deploy to AWS Asia Pacific (Singapore) region

### DISCUSSION THREAD

**Discussion by josedc** [mozilla/hubs#2974] [General Questions & Discussion] [answered]
Hello everyone. I’m in Singapore and I was trying to set up Hubs Cloud then realised only 5 regions are supported. I think the closest region to me is Asia Pacific (Tokyo) but the CF script failed both my attempts. I finally got it running through US West (Oregon) but it’s not viable long term. Is there some reason the CF script couldn’t use Asia Pacific (Singapore) region?

**Comment by michela** [comment]
I'd like to deploy to the Sydney region so interested in response to this. My understanding is that AWS would require explicit setup of region specific AMIs so it might simply be a matter of Mozilla prioritising its resources.

**Comment by robinkwilson** [comment, marked as answer]
Right now, the only regions to deploy HC on aws are the 5 regions, it's currently not on our roadmap to increase that number. 

AFAIK, HC requires specific services that aren't available in all regions which is why these 5 were chosen since they have a spread across the world.

@josedc That's strange that the same deploy didn't work for you in Tokyo but did in Oregon, can you make a github discussion thread about what you saw? And mention me in a comment?

**Reply by josedc** [reply]
Thanks for that ... I started a new discussion with info about the CF errors I saw trying to deploy to Tokyo.

**Comment by DoisKoh** [comment]
I'm in Singapore, using Tokyo region and my CF initialized just fine.

**Reply by josedc** [reply]
Thanks for that ... I documented my CF errors in a new discussion. Hopefully I can figure out what the issue is.
